comunicatii/src/main/java/code4romania/czl/scrapers/comunicatii/ComunicatiiBean.java

```diff
@@ -14,202 +13,199 @@
-        }
-
-        public void setUrl(String url) {
-            this.url = url;
-        }
-    }
-
-    public String getIdentifier() {
-        return identifier;
-    }
-
-    public void setIdentifier(String identifier) {
-        this.identifier = identifier;
-    }
-
-    public String getTitle() {
-        return title;
-    }
-
-    public void setTitle(String title) {
-        this.title = title;
-    }
-
-    public String getType() {
-        return type;
-    }
-
-    public void setType(String type) {
-        this.type = type;
-    }
-
-    public String getInstitution() {
-        return institution;
-    }
-
-    public void setInstitution(String institution) {
-        this.institution = institution;
-    }
-
-    public String getDate() {
-        return date;
-    }
-
-    public void setDate(String date) {
-        this.date = date;
-    }
-
-    public void setDate(String date, String fmt){
-
-        DateTimeFormatter in = DateTimeFormatter.ofPattern(fmt);
-        DateTimeFormatter out = DateTimeFormatter.ofPattern("2017-03-08");
-        try {
-            LocalDate localDate = LocalDate.parse(date, in);
-
-            this.date = localDate.format(out).toString();
-        }catch (Exception e){
-            e.printStackTrace();
-        }
-     }
-
-
-    public String getDescription() {
-        return description;
-    }
-
-    public void setDescription(String description) {
-        this.description = description;
-    }
-
-    public String getFeedback_days() {
-        return feedback_days;
-    }
-
-    public void setFeedback_days(String feedback_days) {
-        this.feedback_days = feedback_days;
-    }
-
-   public Contact getContact() {
-       return contacts;
-   }
-
-    public void setContact(Contact contact) {
-        this.contacts = contact;
-    }
-    public List<Document> getDocuments() {
-        if (documents == null){
-            documents = new ArrayList<Document>();
-        }
-        return documents;
-    }
-
-    public void setDocuments(List<Document> documents) {
-        this.documents = documents;
-    }
-
-    public void addDocument(String url, String type) {
-        Document doc = new Document();
-        doc.setType(type);
-        doc.setUrl(url);
-        getDocuments().add(doc);
-    }
-
-    public String toJson() {
-        try {
-            String result = new ObjectMapper()
-                    .writeValueAsString(this);
-            return result;
-        }catch (JsonProcessingException e) {
-            e.printStackTrace();
-        }
-
-        return "{}";
-    }
-
+	String identifier;
+	String title;
+
+	String type = " ";
+	String institution = "comunicatii";
+	String date;
+	String description;
+	String feedback_days = "0";
+	// String contact = "[\"email\":\"propuneri@comunicatii.gov.ro\"]";
+	Contact contacts = new Contact();
+	List<Document> documents;
+
+	static class Document {
+		String type;
+		String url;
+
+		@Override
+		public String toString() {
```

Comment:
Exista o utilitate numita `ToStringBuilder` in commons-lang3 care transforma `toString()` intr-o singura linie.